how-to.v2/use-a-sidecar-app-basic/platform/client/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+	"compilerOptions": {
+		"target": "es2021",
+		"module": "ES2020",
+		"sourceMap": true,
+		"rootDir": "./src",
+		"outDir": "build",
+		"skipLibCheck": true,
+		"strict": true,
+		"esModuleInterop": true,
+		"forceConsistentCasingInFileNames": true,
+		"moduleResolution": "node",
+		"types": ["./fin"]
+	},
+	"include": ["./src/**/*.ts", "fin.d.ts"]
```

Comment:
not sure why fin.d.ts is in here, if anything the types on line 13 should say ["./types"] and the fin.d.ts file should be in a types folder, to match other repos

Reply:
updated